### ISSUE TITLE
[BUG]/User/xxxx/.pub-cache/hosted/pub.flutter-io.cn/fijkplayer-0.11.0/android/src/main/java/com/befovy/fijkplayer/FijkPlugin.java:57: 警告: [deprecation] PluginRegistry中的Registrar已过时 import io.flutter.plugin.common.PluginRegistry.Registrar;

### ISSUE THREAD

**Issue by jeckni** [befovy/fijkplayer#638]
/Users/nizhangfu/.pub-cache/hosted/pub.flutter-io.cn/fijkplayer-0.11.0/android/src/main/java/com/befovy/fijkplayer/FijkPlugin.java:57: 警告: [deprecation] PluginRegistry中的Registrar已过时
import io.flutter.plugin.common.PluginRegistry.Registrar;
                                              ^
/Users/xxxx/.pub-cache/hosted/pub.flutter-io.cn/fijkplayer-0.11.0/android/src/main/java/com/befovy/fijkplayer/FijkPlugin.java:84: 警告: [deprecation] PluginRegistry中的Registrar已过时
    private Registrar mRegistrar;
            ^
/Users/xxxx/.pub-cache/hosted/pub.flutter-io.cn/fijkplayer-0.11.0/android/src/main/java/com/befovy/fijkplayer/FijkPlugin.java:104: 警告: [deprecation] PluginRegistry中的Registrar已过时
    public static void registerWith(Registrar registrar) {
                                    ^
/Users/xxxx/.pub-cache/hosted/pub.flutter-io.cn/fijkplayer-0.11.0/android/src/main/java/com/befovy/fijkplayer/FijkPlugin.java:229: 警告: [deprecation] PluginRegistry中的Registrar已过时
    private void initWithRegistrar(@NonNull Registrar registrar) {
                                            ^
/Users/xxxx/.pub-cache/hosted/pub.flutter-io.cn/fijkplayer-0.11.0/android/src/main/java/com/befovy/fijkplayer/FijkPlugin.java:570: 警告: [deprecation] AudioManager中的abandonAudioFocus(OnAudioFocusChangeListener)已过时
            audioManager.abandonAudioFocus(this);
                        ^
5 个警告

提示有5个警告，我该如何才能避免运行爆出这五个警告呀。

**Comment by yeshu-cn** [comment]
3.29.0已经删除了这几个api，编译不过了

**Comment by wupengbigMarker** [comment]
> 3.29.0已经删除了这几个api，编译不过了

Did u solve this issue

**Comment by yeshu-cn** [comment]
> > 3.29.0已经删除了这几个api，编译不过了
> 
> Did u solve this issue

No, I just worked around it by downgrading the Flutter SDK to 3.27.4.

**Comment by shiponmvn** [comment]
Is there any solution without downgrading the SDK version? I need to use version 3.29.0.

**Comment by zp-techie** [comment]
> Is there any solution without downgrading the SDK version? I need to use version 3.29.0.

May I ask if you have solved this
